src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonTableViewCell.swift

```diff
@@ -0,0 +1,162 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class HealthCertifiedPersonTableViewCell: UITableViewCell, ReuseIdentifierProviding {
+	
+	// MARK: - Overrides
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		setupView()
+		setupAccessibility()
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override func setHighlighted(_ highlighted: Bool, animated: Bool) {
+		super.setHighlighted(highlighted, animated: animated)
+
+		cardView.setHighlighted(highlighted, animated: animated)
+	}
+
+	// MARK: - Internal
+
+	func configure(with cellModel: HealthCertifiedPersonCellModel) {
+		titleLabel.attributedText = cellModel.attributedText
+		nameLabel.text = cellModel.name
+		gradientView.type = cellModel.backgroundGradientType
+		// add a placeholder image and process the QRCode image in background to smooth scrolling a bit
+		qrCodeImageView.image = placeHolderImage
+		DispatchQueue.global(qos: .userInteractive).async {
+			let qrCodeImage = UIImage.qrCode(
+				with: cellModel.certificate.base45,
+				encoding: .utf8,
+				size: CGSize(width: 280, height: 280),
+				qrCodeErrorCorrectionLevel: .medium
```

Comment:
According to [the tech spec](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/48b557b918634a6c87ef6de615734bb6613331c4/docs/spec/dgc-overview-client.md#generating-a-qr-code-for-a-dgc) (4.) we should use `.quartile` for digital COVID certificates

Reply:
done